### PR TITLE
Update to latest Substrate

### PR DESCRIPTION
**Pull request by tomaka**

### CODE REVIEW

File: service/src/chain_spec.rs [hunk 1]
```diff
@@ -125,6 +125,7 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 			minimum_validator_count: 4,
 			stakers: initial_authorities.iter().map(|x| (x.0.clone(), x.1.clone(), STASH, StakerStatus::Validator)).collect(),
 			invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect(),
+			.. Default::default()
```
Comment (tomaka) [inline service/src/chain_spec.rs]: Note: that's the `force_era` field of https://github.com/paritytech/substrate/pull/3374

File: cli/src/lib.rs [hunk 2]
```diff
@@ -143,7 +143,7 @@ fn run_until_exit<T, C, W>(
 	let _telemetry = service.telemetry();
 
 	let work = worker.work(&*service, Arc::new(executor));
-	let _ = runtime.block_on(service.select(work));
+	let _ = runtime.block_on(service.map_err(|_| ()).select(work));
```
Comment (bkchr) [inline cli/src/lib.rs]: Is service now returning a different error type than `()`?

Comment (tomaka) [reply cli/src/lib.rs]: Yes, a `ServiceError` apparently: https://github.com/paritytech/substrate/pull/3353/files#diff-d2b444bb5d594e411359291f8be4fe56

Comment (bkchr) [reply cli/src/lib.rs]: Why do we not want to at least log this error?

File: service/src/chain_spec.rs (hunk 1 above)
Comment (shawntabrizi) [reply service/src/chain_spec.rs]: I think the intention is that this one is supposed to be `Forcing::ForceNone`, but you should get @gavofyork to confirm

**Comment by tomaka** [comment]
CI fails because of the missing `force_eras` field. Since I don't know which values to use, I'm going to leave that to competent people.

**Comment by bkchr** [comment]
Superseded by: https://github.com/paritytech/polkadot/pull/362

**Pull request closed without merging**